.appveyor.yml

```diff
@@ -14,18 +14,13 @@ environment:
   global:
     ICU_CI_CACHE: c:\icu-ci-cache
     CYG_URL: https://cygwin.com/setup-x86_64.exe
-    CYG_MIRROR: http://cygwin.mirror.constant.com
+    CYG_MIRROR: http://ctm.crouchingtigerhiddenfruitbat.org/pub/cygwin/circa/64bit/2019/02/13/045332
```

Comment:
What is this URL and where did you get it?

Reply:
This URL points to an archive repository of the last release of the 64-bit version of Cygwin 2.11.2-1, which was the last release before version 3.0.0 came out.
I got the URL by following the instructions on the official Cygwin page (https://cygwin.com/) for old versions, which directs you to the "Cygwin Time Machine" (http://www.crouchingtigerhiddenfruitbat.org/Cygwin/timemachine.html).
From there you can pick either 32-bit or 64-bit, and then find a listing of URLs for various prior versions of Cygwin.

I think I'll add some comments to the AppVeyor file about this as well, so that others (in the future) would know as well.